packages/runtime/src/runtime/hooks/__tests__/useActionGroup.test.tsx

```diff
@@ -164,6 +190,120 @@ test("fetch multiple APIs", async () => {
   });
 });
 
+test("fetch API cache response for cache expiry", async () => {
```

Comment:
Move these tests to the useInvokeApi test suite. Also add a test that asserts if the cacheExpiry is not present, then the response is not cached.